File: multilevelinheritance.py
class Vehicle:
    def __init__(self):
        self.model = ""
        self.price = 0

    def set_brand(self):
        print("Brand: Toyota")


class Car(Vehicle):
    def __init__(self, model):
        super().__init__()
        self.model = model


class MicroCar(Car):
    def __init__(self, model):
        super().__init__(model)


obj = Car("Car")
obj.set_brand()

obj2 = MicroCar("Cycle")
obj2.set_brand()

File: multilevelinheritance.cpp
#include<iostream>
using namespace std;

class Vehicle{
public:
    string model;
    int price;

    void setBrand(){
        cout<<"Brand : "<<"Toyota"<<endl;
    }
};

class Car:public Vehicle{
public:
    Car(string model) : Vehicle() { 
        this->model=model;    
    }
};

class micro_car:public Car{
public:
    micro_car(string model) : Car(model) {
        
    } 
};

int main(){
    Car obj("Car");
    obj.setBrand();

    micro_car obj2("Cycle");
    obj2.setBrand(); 

    return 0;
}
